libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.scss

```diff
@@ -0,0 +1,10 @@
+@import "~fundamental-styles/dist/resizable-card-layout";
+
+.fd-resizable-card-expansion {
+    transition-duration: 350ms;
+    animation-timing-function: ease-in-out;
```

Comment:
are these values taken from the specs?

Reply:
these values are not provided in specs. have asked designers. but currently they do not have these values and this component is not in their priority for now. for unblocking the development  i have used these values, which come close to achieving the required behaviour. 
i have asked for the values again. if answers come, i will update it.